bower.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "angular-evaporate",
+  "homepage": "https://github.com/uqee/angular-evaporate",
+  "description": "AngularJS module for the EvaporateJS library",
+  "authors": [
+    "Denis Zhbankov <denis.zhbankov@gmail.com>"
+  ],
+  "keywords": [
+    "upload",
+    "resumable upload",
+    "evaporate",
+    "angular"
+  ],
+  "version": "2.1.2",
+  "_release": "2.1.2",
+  "_resolution": {
+    "type": "version",
+    "tag": "v2.1.2",
+    "commit": "c332e5e4830b695a33cfc8db16420188cc7ab13d"
+  },
+  "_source": "https://github.com/uqee/angular-evaporate.git",
+  "_target": "*",
+  "_originalSource": "angular-evaporate",
+  "dependencies": {
+    "angular": "^1.5.8",
+    "evaporatejs": "^1.5.2"
+  },
+  "devDependencies": {
```

Comment:
I guess, the contents of `devDependencies` cannot be simply copied form the `package.json`, because some packages are not represented in bower (e.g. `morgan` or `serve-static`). It's probably better to just remove the `devDependencies` property at all and utilize npm for the dev purpose.